datacatalog/cloud-client/requirements.txt

```diff
@@ -0,0 +1 @@
+google-cloud-datacatalog
```

Comment:
We usually pin the version number in our sample code. (There's a bot that comes around to update these files to the latest version)

Reply:
Done.